### ISSUE TITLE
LaTeX input: \% is interpreted as literal \%, not as a command printing a %. 

### ISSUE THREAD

**Issue by gunterkoenigsmann** [mathjax/MathJax#1473]
Dear all,

One half-way minimal example:
```
<!DOCTYPE HTML>
<HTML>
  <HEAD>
    <script type="text/x-mathjax-config">
      MathJax.Hub.Config({
      displayAlign: "left",
      context: "MathJax"
      })
    </script>
    <script type="text/javascript"
	    src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML">
    </script>
    <link rel="stylesheet" type="text/css" href="TableForm_htmlimg/TableForm.css"/>
  </HEAD>
  <BODY>
    \[
    \tag{\%{}t1}\label{t1} \begin{align}1 & 2\\
    3 & 4\end{align}
    \]
  </BODY>
</HTML>
```
The label that is shown in the browser should be %t1, not \%t1.

**Comment by dpvc** [comment]
The contents of the `\tag{}` command is text-mode material.  MathJax does not process macros in text-mode, only math-mode, so you should use `\tag{%t1}` in this case.  (Your `\tag{%{}t1}` produces `\%{}t1`, not just `\%t1`.)  See #790, #569, #346, #243, and others.
